main/autogen/autogen.cc

```diff
@@ -674,4 +687,34 @@ void ParsedFile::classlist(core::Context ctx, vector<string> &out) {
     }
 }
 
+void ParsedFile::subclasses(core::Context ctx, map<string, set<string>> &out) {
+    // Ignore files in dirs we don't care about
+    if (path.find("test/") == 0 || path.find("scripts/") == 0 || path.find("lib/identification/scripts/") == 0 ||
+        path.find("ruby_benchmarks/") == 0 || path.find("/test/") != string_view::npos ||
+        path.find("/benchmark/") != string_view::npos) {
+        // Ignoring 'test/' also ignores the Sorbet CLI test input files :(
+        if (!(path.find("test/cli/autogen-subclasses") == 0)) {
+            return;
+        }
+    }
+
+    auto nameToString = [&](const core::NameRef &nm) -> string { return nm.data(ctx)->show(ctx); };
```

Comment:
please write it inline and duplicate it. We try to not store capturing lambdas.

Reply:
So the capturing clause can actually just be `[&ctx]` (they only need the `core::Context`). Does that change how you feel about storing it? I'm fine either way.